src/current/v25.4/known-limitations.md

```diff
@@ -12,25 +12,26 @@ docs_area: releases
 
 This section describes newly identified limitations in CockroachDB {{ page.version.version }}.
 
-### Row-level security filtering
+### View support
 
-{% include {{ page.version.version }}/known-limitations/rls-update-set-where-returning.md %}
+{% include {{ page.version.version }}/known-limitations/view-limitations.md %}
 
-### DistSQL
+### User-defined functions
 
-{% include {{ page.version.version }}/known-limitations/distsql-heterogeneous-endianness.md %}
+- User-defined functions are not currently supported in partial index predicates. [#155488](https://github.com/cockroachdb/cockroach/issues/155488)
+- Views cannot reference a UDF that contains mutation statements (`INSERT`, `UPDATE`, `UPSERT`, `DELETE`). [#151686](https://github.com/cockroachdb/cockroach/issues/151686)
 
-### Multi-region
+### Stored procedures
 
-{% include {{ page.version.version }}/known-limitations/enforce-home-region-limitations.md %}
+- Pausable portals are not supported with `CALL` statements for stored procedures. [#151529](https://github.com/cockroachdb/cockroach/issues/151529)
 
-### Geospatial
+### Mixed-isolation workloads
 
-{% include {{ page.version.version }}/known-limitations/geospatial-heterogeneous-architectures.md %}
+- Mixed-isolation-level workloads must enable foreign-key check locking for `SERIALIZABLE` transactions to avoid race conditions. [#151663](https://github.com/cockroachdb/cockroach/issues/151663)
```

Comment:
Same comment here about linking directly to comment https://github.com/cockroachdb/cockroach/issues/151663#issuecomment-3222083180